### PR TITLE
Add `TestExecutionStatus` to admin

### PR DESCRIPTION
**Pull request by asankov**
Refs #236

- Add `colorfield` to `INSTALLED_APPS`
- Change `TestExecutionStatus::color` from `models.CharField` to
  `colorfield.ColorField`

### CODE REVIEW

**Comment by asankov** [comment]
@atodorov can you take a look at the failing build. the `build-for-pypi` and `test-docker-image` jobs are failing with:
```
ERROR: No matching distribution found for django-colorfield==0.1.15 (from kiwitcms)
```
but the other jobs seem to be ok with finding the new dependency

File: requirements/base.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ django-attachments==1.5
 django-contrib-comments==1.9.2
 django-modern-rpc==0.12.0
 django-simple-history==2.8.0
+django-colorfield==0.1.15
```
Comment (atodorov) [inline requirements/base.txt]: There is no wheel package for this so add to tarballs.txt so Ci can pass.

**Comment by asankov** [comment]
@atodorov this should be ready for merging

**Comment by atodorov** [comment]
> @atodorov this should be ready for merging

It is not. This PR is blocked by #1258 because merging will open the possibility to edit TE statuses without users being fully able to make use of them. The admin portion also needs: 
- show id, color & weight in list views
- in show/edit mode display the names in English, not translated into browser language (b/c that may lead to further confusion and errors down the road, haven't tested it yet)
- need to check admin permissions and very likely we need a data migration to update permission labels for group Tester so that these newly exposed objects can be edited by users (haven't tested this yet).
- good documentation about this change and its general behavior (positive, negative and neutral statuses). Then we close #236. 

These other items are relatively easy to do and should be done shortly after this has been merged. For now this is BLOCKED.

File: tcms/testruns/models.py [hunk 2]
```diff
@@ -235,11 +236,15 @@ def stats_executions_status(self, statuses=None):
 
 class TestExecutionStatus(TCMSActionModel):
 
+    class Meta:
+        # used in the admin view
+        verbose_name_plural = "Test execution statuses"
```
Comment (atodorov) [inline tcms/testruns/models.py]: Needs to be marked for translation as well.

**Comment by asankov** [comment]
> show id, color & weight in list views

Done

> need to check admin permissions and very likely we need a data migration to update permission labels for group Tester so that these newly exposed objects can be edited by users (haven't tested this yet).

My local user is Tester, and I am seeing the new admin view (and modify the statuses).
Screen Shot 2020-01-31 at 23 48 34
Screen Shot 2020-01-31 at 23 49 13

> good documentation about this change and its general behavior (positive, negative and neutral statuses). Then we close #236.

Okay. I suppose you mean the documentation that we publish in `readthedocs`? Maybe somewhere in `docs/source/guide/testrun.rst`?

**Pull request merged**